dynamo/tools/connectivity.py

```diff
@@ -81,6 +85,18 @@ def knn_to_adj(knn_indices, knn_weights):
 
 
 def get_conn_dist_graph(knn, distances):
+    """Compute connection and distance sparse matrices
+
+    Parameters
+    ----------
```

Comment:
let us indent knn ... like other documentation in dynamo  

also add space above Return